app/src/main/java/cz/muni/fi/pv256/movio2/uco374585/MovieDetailFragment.java

```diff
@@ -0,0 +1,80 @@
+package cz.muni.fi.pv256.movio2.uco374585;
+
+import android.graphics.drawable.Drawable;
+import android.os.Bundle;
+import android.app.Fragment;
+import android.support.v4.content.res.ResourcesCompat;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.nostra13.universalimageloader.core.ImageLoader;
+import com.nostra13.universalimageloader.core.ImageLoaderConfiguration;
+
+import java.text.SimpleDateFormat;
+
+import cz.muni.fi.pv256.movio2.uco374585.Models.Movie;
+
+public class MovieDetailFragment extends Fragment {
+
+    private Movie movie;
+    ImageLoader imageLoader;
+
+    public MovieDetailFragment() {
+        // Required empty public constructor
+    }
+
+    public static MovieDetailFragment newInstance(Movie movie) {
+        MovieDetailFragment fragment = new MovieDetailFragment();
+        Bundle bundle = new Bundle();
+        bundle.putParcelable("movie", movie);
+        fragment.setArguments(bundle);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        if (getArguments() != null) {
+            movie = getArguments().getParcelable("movie");
+        }
+        imageLoader = ImageLoader.getInstance();
+        imageLoader.init(ImageLoaderConfiguration.createDefault(getContext()));
```

Comment:
getContext je podporovane az od API level 23, to je prilis vysoko. 
bud je potrebne zvolit iny pristup, alebo urobit if, a pre API>23 pouzit tento pristup, a pre API level<23 iny